### DISCUSSION TITLE
Upgrade from 9.05 to 9.10.2 - System.Web.Http version error

### DISCUSSION THREAD

**Discussion by mp9007** [dnnsoftware/Dnn.Platform#5037] [Q&A]
Hi, when copy/pasting upgrade files into my website folder and then going the the /install/upgradwizard.aspx, I have many errors in my log. 
I guess it is because in my bin folder, I have System.Web.Http version 5.2.3 and not 5.2.7. 
But I dont know how to update this file. I did update extensions I had : 2sxc and CKEditor.
Why can't I find the newer version of System.Web.Http in the upgrade /bin folder ?

Thanks.

Here is the error I get. This error is repeated for many other services.

[Thread:40][ERROR] DotNetNuke.Web.DependencyInjectionInitialize - Unable to configure services for DotNetNuke.Modules.CoreMessaging, Version=9.5.0.0, Culture=neutral, PublicKeyToken=null, see exception for details 
**LoaderException: Could not load file or assembly 'System.Web.Http, Version=5.2.7.0, Culture=neutral,** PublicKeyToken=31bf3856ad364e35' or one of its dependencies. The located assembly's manifest definition does not match the assembly reference. (Exception from HRESULT: 0x80131040)
System.Reflection.ReflectionTypeLoadException: Unable to load one or more of the requested types. Retrieve the LoaderExceptions property for more information.
   at System.Reflection.RuntimeModule.GetTypes(RuntimeModule module)
   at System.Reflection.Assembly.GetTypes()
   at DotNetNuke.DependencyInjection.Extensions.TypeExtensions.SafeGetTypes(Assembly assembly, ILog logger)

**Comment by mitchelsellers** [comment]
Please follow the suggested upgrade path as outlined in the documentation. An intermediate upgrade step can prevent these issues.

**Reply by mp9007** [reply]
I followed the suggested upgrade path. I even did multiple other steps than the suggested upgrade path because I had a lot of issues.

I originally had version 9.02. I did : 
 - 9.02 to 9.03
 - 9.03 to 9.04
 - 9.04 to 9.05

I then tried 
- 9.05 to 9.06, same error
- 9.05 to 9.06.2, same error
- 9.05 to 9.10.2, same error

![image](https://user-images.githubusercontent.com/38858370/156606045-18add561-e1a4-4c6b-b745-04b916b3ff43.png)
